lambda/readFacility/index.js

```diff
@@ -80,6 +93,33 @@ const parkVisible = async function (park, isAdmin) {
   }
 };
 
+const getReservationCounts = async function (parkName, facilityName) {
+  // Gets the reservation counts for a facility and simplifies the results
+  const resCountPK = `rescount::${parkName}::${facilityName}`;
+
+  // query DynamoDB
+  let queryObj = {
+    TableName: TABLE_NAME,
+    ExpressionAttributeValues: {}
+  };  
+  queryObj.ExpressionAttributeValues[':pk'] = { S: resCountPK };
+  const today = formatISO(new Date(), { representation: 'date' });
+  // only reservations on or after today are included
```

Comment:
I'm not sure this is taking into account the right 'today' depending on the time that this runs.

Reply:
I changed the implementation to pure JS instead of using a date library. There is an open bug on formatISO, and it wasn't clear if the behaviour will be changing in the future. https://github.com/date-fns/date-fns/issues/2151

Reply:
This has been simplified to use date-fns format() 
I also fixed another date issue on parks-reso-public so it will set the pass dates consistently in DynamoDB.
https://github.com/bcgov/parks-reso-public/pull/136